### PR TITLE
Update for pr11

### PR DESCRIPTION
**Pull request by supercrafter100**
This pull requests modifies the smarty files & how the module itself works to support the new staffcp layout and have some internal tweaks that rendered it useless. The template files are likely not great at all but it works. (I barely know anything about php & smarty so you may notice that)

It does seem to work on my testing website & server with the latest websend plugin that is on samerton his [github ](https://github.com/samerton/Websend/releases/tag/v2.5.11)

### CODE REVIEW

File: upload/modules/Websend/pages/panel/websend.php [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 // Can the user view the panel?
 if($user->isLoggedIn()){
-	if(!$user->canViewACP()){
```
Comment (tadhgboyle) [inline upload/modules/Websend/pages/panel/websend.php]: Using handlePanelPageLoad() can replace line 13-28 (might be wrong on exact lines-on mobile). Check out other panel pages in the core module to see what I mean

File: upload/modules/Websend/pages/panel/websend.php [hunk 2]
```diff
@@ -132,6 +135,10 @@
 			$port = (isset($_POST['port']) ? (int)$_POST['port'] : 4445);
 			$password = (isset($_POST['password']) ? $_POST['password'] : $ws_config['websend_password']);
 
+            WSHook::setWSHost($adress);
```
Comment (tadhgboyle) [inline upload/modules/Websend/pages/panel/websend.php:138]: $address is spelt wrong

File: upload/modules/Websend/pages/panel/websend.php (hunk 1 above)
Comment (supercrafter100) [reply upload/modules/Websend/pages/panel/websend.php]: Fixed in e74ef8e.

File: upload/modules/Websend/pages/panel/websend.php (hunk 2 above)
Comment (supercrafter100) [reply upload/modules/Websend/pages/panel/websend.php:138]: Fixed in 31e2894.

**Pull request closed without merging**